

# mesaage for misc commands

uint8 RESET_THRUSTERS = 0
uint8 REFRESH = 1
uint8 SHUT_DOWN = 2

uint8 command

